src/autocomplete.ts

```diff
@@ -70,6 +93,15 @@ export default class Autocomplete {
     this.results.removeEventListener('combobox-commit', this.onCommit)
   }
 
+  handleClear(event: Event): void {
+    event.preventDefault()
+
+    this.input.value = ''
+    this.container.value = ''
+    this.input.focus()
```

Comment:
We may want a `change` event here to allow for consumers of this element to be notified when the input changes content.

```suggestion
    this.input.focus()
    this.input.dispatchEvent(new Event('change'))
```